firefox-ios/Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -63,8 +63,7 @@ class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
         containerView.addSubview(logoTextImage)
         contentView.addSubview(containerView)
 
-        // TODO: Felt Privacy - Private mode in Redux to follow
-        let isiPadAndPrivate = UIDevice.current.userInterfaceIdiom == .pad && false
+        let isiPadAndPrivate = UIDevice.current.userInterfaceIdiom == .pad
```

Comment:
Remove `AndPrivate` from the name as it's no longer reflective of that. :)

Reply:
oo missed that, thanks for the catch! updated :)